Contributing.md

```diff
@@ -0,0 +1,99 @@
+# Contributing to RealtimeSanitizer
+
+This document describes some helpful tips and tricks for contributing to RealtimeSanitizer. 
+
+RealtimeSanitizer is a part of LLVM, meaning that when in conflict, the two following guides take precedence over this one:
+
+1. [Contributing to LLVM](https://llvm.org/docs/Contributing.html)
+2. [LLVM Coding Standards](https://llvm.org/docs/CodingStandards.html)
+
+We highly recommend reading these two guides, as they have a lot of things not covered in this document. 
+
+The goal of this document is to be a minimal addition to the above, providing specific advice for developing RealtimeSanitizer.
+
+# Building and running tests
```

Comment:
I recommend adding CMake(version 3.20 or higher) to this list of dependencies/recommended packages too. This is the first big project I'm setting up on a new laptop, so I had to install that in addition to ninja and ccache.

Reply:
I have mixed feelings about this one. I don't think it would hurt to add it, but I also think that it is "implicitly required" or covered by the other guides. Meaning "if a user tries to build without cmake, it will be dead obvious that they need it right away".

Why ccache and ninja are here is because technically you could build without them, but they just make your life a lot easier as a contributor - speeding things up immensely. This hidden "tip/trick" is what I think this guide can be really good, as opposed to covering things that are covered in other guides, or pop up in an immediate/obvious way.

What do you think? I realize this is a subjective and arbitrary line I'm drawing in the sand :)

Reply:
That logic makes sense to me! I'm good with excluding the cmake version here then.